src/components/views/mechanisms/MechSimple.js

```diff
@@ -5,73 +5,29 @@ import NodeSelection from "./NodeSelection";
 import { PortWidget } from "@projectstorm/react-diagrams";
 
 const styles = () => ({
-  root: {
-    border: 'solid 0.0625rem',
-    borderRadius: '50%',
-    width: '10rem',
-    height: '10rem',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-    flexDirection: 'column',
-    position: 'relative',
-
-    '& img': {
-      marginBottom: '0.25rem',
-    },
-
-    '& p': {
-      fontWeight: 500,
-      fontSize: '0.8125rem',
-      lineHeight: '1.25rem',
-      letterSpacing: '-0.005rem',
-      margin: 0,
-    },
-  },
 });
 
 class MechSimple extends React.Component {
   render() {
-    const { classes, node, node: { options }, engine, changeVisibility } = this.props;
+    const { classes, model, model: { options }, engine, changeVisibility } = this.props;
 
-    const circlePortStyle = {
-      width: "12px",
-      height: "12px",
-      margin: "2px",
-      borderRadius: "4px",
-      background: "darkgray",
-      cursor: "pointer",
+    const customNodeStyles = {
+      background: options.variant.backgroundColor,
```

Comment:
@vidhya-metacell do we need this?

Reply:
@zsinnema Yes, so the nodes can have any color options, they can be orange, green whatever. So from the psyneulink or any project that would use the theme, we can easily pass to the node what color option it should follow, also we can add many more color options according to the project need. Because the node styling is in meta-diagram, this gives us a control over the color variants from the project it is being used on.